README.md

```diff
@@ -143,31 +143,34 @@ bash ~/Docker-Provider/scripts/build/linux/install-build-pre-requisites.sh
 
 > Note: If you are using WSL2, ensure `Docker for windows` running with Linux containers mode on your windows machine to build Linux agent image successfully
 
+> Note: format of the imagetag will be `ci<release><MMDDYYYY>`. possible values for release are test, dev, preview, dogfood, prod etc. Please use MCR urls while building internally.
+
+Preferred Way: You can build and push images for multiple architectures. This is powered by docker buildx
+Directly use the docker buildx commands (the MCR images can be found in our internal wiki to be used as arguments)
+```
+# multiple platforms
+cd ~/Docker-Provider
+docker buildx build --platform linux/arm64/v8,linux/amd64 -t <repo>/<imagename>:<imagetag> --build-arg IMAGE_TAG=<imagetag> --build-arg UBUNTU_BASE_IMAGE=<ubuntuimage> --build-arg GOLANG_BASE_IMAGE=<golangimage> -f kubernetes/linux/Dockerfile.multiarch --push .
```

Comment:
considering the upcoming mariner change, probable we should change the naming generic so that we dont need to make the changes to all these files again, right?

Reply:
> you mean from UBUNTU_BASE_IMAGE to BASE_IMAGE?

> Yes. And also,  other related question, when we move to Mariner, container security guidance not applicable for Mariner base image as this image will be pulled from MCR, right?